ethpm_types/contract_type.py

```diff
@@ -1,5 +1,6 @@
+from collections.abc import Callable, Iterable
```

Comment:
same here, is it suggested to import Callable from here instead as of some python version?

Reply:
This is correct. Basically anything that is an alias in typing is now deprecated.

Reply:
ahhhh, okay I get it now